tanner/tests/test_config.py

```diff
@@ -41,7 +42,8 @@ def setUp(self):
 
         self.invalid_config_path = '/random/random_name'
 
-    def test_set_config_when_file_exists(self):
+    @mock.patch('tanner.tests.test_config.config')
```

Comment:
Is it required?

Reply:
Yes absolutely :)

Reply:
But it was not needed previously.

Reply:
We should patch `config` here to limit the scope to this test. Because next tests uses changed settings which should not happen. This is the reason why `test_api` works.

Reply:
Do you have some other fix in mind?

Reply:
I don't understand this code, patching is used to point one object to another. And also since you have changed the value of redis port in setUp method so I guess no more changes are required.

Reply:
Patching is also used to limit the scope of the test. Other tests are also affected in `dorks_manager` which checks for certain function to be called so only changing port settings won't work. I guess patching it here does no harm to `test_config`.